awscli/data/codebuild/2016-10-06/completions-1.json

```diff
@@ -1,14 +1,114 @@
 {
   "version": "1.0",
   "resources": {
+    "BuildBatche": {
+      "operation": "ListBuildBatches",
+      "resourceIdentifier": {
+        "id": "ids[]"
+      }
+    },
     "Build": {
       "operation": "ListBuilds",
       "resourceIdentifier": {
         "id": "ids[]"
       }
+    },
+    "SourceCredential": {
+      "operation": "ListSourceCredentials",
+      "resourceIdentifier": {
+        "arn": "sourceCredentialsInfos[].arn",
+        "serverType": "sourceCredentialsInfos[].serverType",
+        "authType": "sourceCredentialsInfos[].authType"
+      }
     }
   },
   "operations": {
+    "DeleteBuildBatch": {
+      "id": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "BuildBatche",
+            "resourceIdentifier": "id"
+          }
+        ]
+      }
+    },
+    "DeleteReport": {
+      "arn": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "SourceCredential",
```

Comment:
That's definitely not the correct resource we want to be using. I'd assume that we would want to generate a `Report` resource and use its arn?